src/plugins/util/i18n.ts

```diff
@@ -1,9 +1,46 @@
 import Map from '@dojo/shim/Map';
-import { CallExpression, Identifier, Program, VariableDeclaration, VariableDeclarator } from 'estree';
+import { CallExpression, Expression, Identifier, Program, SequenceExpression, SpreadElement, VariableDeclaration, VariableDeclarator } from 'estree';
 import * as path from 'path';
 import { getBasePath, isRelative, mergeUnique } from './main';
 import { extractArrayValues, getNextItem, isShadowing } from './parser';
 
+/**
+ * @private
+ * Determines whether the provided expression is a CallExpression.
+ */
+function isCallExpression(expression: any): expression is CallExpression {
+	return expression && expression.type === 'CallExpression';
+}
+
+/**
+ * @private
+ * Determines whether the provided expression is a SequenceExpression.
+ */
+function isSequenceExpression(expression: any): expression is SequenceExpression {
+	return expression && expression.type === 'SequenceExpression';
+}
+
+/**
+ * @private
+ * Resolves the callee and arguments for the provided expression.
+ *
+ * @param expression An AST expression node.
+ * @return An object containing the callee and arguments if the expression is a call expression.
+ */
+function getCallExpressionData(expression?: any): { callee?: Identifier, args?: Array<Expression | SpreadElement> } {
+	if (isSequenceExpression(expression)) {
+		const callExpressions = expression.expressions.filter(expression => isCallExpression(expression)) as CallExpression[];
+		expression = callExpressions[0];
+	}
+
+	if (!expression) {
+		return {};
+	}
+
+	const { callee, 'arguments': args } = expression;
```

Comment:
do we need the quotes? would just straight `const { callee, arguments: args } = expression;` work?

Reply:
That would work, it was just an arbitrary personal preference on my part.